src/components/Button/Button.tsx

```diff
@@ -89,27 +89,28 @@ const Button = React.forwardRef<
 export default Button;
 
 const activeStyles = css<ElementProps>`
-  color: ${(props) => props.theme.textlight};
-  background-color: ${(props) => {
-    if (props.link) {
-      return props.theme.layerdark;
+  color: ${({ theme }) => theme.textlight};
+
+  background-color: ${({ color, link, theme }) => {
+    if (link) {
+      return theme.layerdark;
     }
 
-    if (props.color === ButtonColor.Light) {
-      return props.theme.layerlightfaded;
+    if (color === ButtonColor.Light) {
+      return theme.layerlightfaded;
     }
 
-    if (props.color === ButtonColor.Lighter) {
-      return props.theme.layerlighterfaded;
+    if (color === ButtonColor.Lighter) {
+      return theme.layerlighterfaded;
     }
 
-    return props.theme.colorpurplefaded;
+    return theme.colorpurplefaded;
   }};
 
-  ${(props) =>
-    props.link
+  ${({ link, theme }) =>
+    link
       ? `${StyledLinkIcon} > svg path {
-        stroke: ${props.theme.textlight};
+        stroke: ${theme.textlight};
```

Comment:
`stroke: currentColor` and let `color` inherit from the parent component. I would like to update this pattern for SVGs everywhere